ci/hardware_tests/bootstrap.py

```diff
@@ -0,0 +1,22 @@
+import configparser
+import sys
+
+from device.t1 import TrezorOne
+
+
+def main(version: str):
+    config = configparser.ConfigParser()
+    config.read_file(open("hardware.cfg"))
+    t1 = TrezorOne(
```

Comment:
`t1 = TrezorOne(config["t1"])` (see below for `t1`), and then parse out the params you need in the TrezorOne class?

Reply:
It seems a bit clearer passing the particular items, instead of an array to the constructor.

Reply:
Can the config ever contain more (or less) than the exact list of parameters for the `TrezorOne` class?
If the answer is yes, then this maybe makes sense. Otherwise I'd at least go with `TrezorOne(**config["t1"])`
As I see it, you're not passing "an array", but "a configuration".

I mean, you are in full control of the config file, call site, and class definition, and you are explicitly listing the arguments one by one in each of the three places. When you add a fourth argument, you will be touching three places again.

Reply:
but annnyway let's not stop the PR on this, this piece of code is unlikely to need that much care in the foreseeable future

Reply:
Got it. And I agree, let's do it later if needed.